src/lib/stable/ic_apis/msg_arg_data.ts

```diff
@@ -15,12 +15,12 @@
  *   - after a successful inter-canister await
  *   - after a successful inter-canister await from a composite query
  */
-export function msgArgData(): Uint8Array {
+export function msgArgData(): Uint8Array<ArrayBuffer> {
     if (
         globalThis._azleIcStable === undefined &&
         globalThis._azleIcExperimental === undefined
     ) {
-        return new Uint8Array();
+        return new Uint8Array(new ArrayBuffer(0));
```

Comment:
Seriously, this has to be done now???????

Reply:
We can't at least leave off the 0?

Reply:
Hmmm... apparently it doesn't have to be done... maybe I did those fixes out of order and it was a type inference problem or something like that from mismatched return types?